### DISCUSSION TITLE
Any idea why full screen flash needed on dictionary dialog closing?

### DISCUSSION THREAD

**Discussion by comphilip** [koreader/koreader#8922] [Q&A] [answered]
The code trigger full screen flash on dialog closing: https://github.com/koreader/koreader/blob/af7c8cade8929e6c542e9e031656bcb23c3aac4f/frontend/ui/widget/dictquicklookup.lua#L901

Why not
```lua
UIManager:setDirty(self, function()
        return "flashui", self.dict_frame.dimen
    end)
```

**Comment by NiLuJe** [comment, marked as answer]
Because it's necessarily over text content, and a localized flash would leave a localized ghosted region with different contrast and a higher chance of ghosting artifacts on the widget's edges (among other things), especially on older devices ;).
